nodexml/README.md

```diff
@@ -0,0 +1,79 @@
+Node XML Schema
+===============
+
+
+Work-In-Progress
+----------------
+
+This readme, the schema, and the example document/s are all still a
```

Comment:
Ok. I'll just refrain from adding that next time, since I usually don't even push to the branch until it's pretty much ready for review.